tcms/testplans/templates/testplans/mutable.html

```diff
@@ -24,6 +32,18 @@
                 </div>
             </div>
 
+        {% if test_plan %}
+            <div class="form-group">
+                <label for="id_active" class="col-md1 col-lg-1"> {% trans "Active" %}</label>
+                <div class="col-md-1 col-lg-1"> {{ form.is_active }} </div>
+            
+                <div>
+                    <label class="col-md-1 col-lg-1"> {% trans "Author" %}</label>
+                    <span class="col-md-2 col-lg-2"><a href="{% url "tcms-profile" test_plan.author.username %}">{{ test_plan.author.username }}</a></span>
+                </div>
```

Comment:
Remove this section. When editing I don't think we really need to know the author of the document (which we can see when viewing the document). Plus you can't change the Author so why have it on the screen in the first place.